src/components/Picker/Picker.tsx

```diff
@@ -40,7 +39,6 @@ export interface Props {
   theme?: any;
   autoSuggest?: boolean;
   autoSuggestMethods?: object[];
-  itemsList?: object[];
   stateProps?: {chipListState: any, suggestions: any, inputProps: object, value?: any};
```

Comment:
These types are sadly defined, I don't like passing state around as being done so many times in components above.